manifests/service.pp

```diff
@@ -4,18 +4,36 @@
 class candlepin::service {
   assert_private()
 
-  service { 'tomcat':
-    ensure     => running,
-    enable     => true,
-    hasstatus  => true,
-    hasrestart => true,
-  }
+	if $candlepin::container {
+		ensure_packages(['podman'])
+
+		file { '/etc/sysconfig/candlepin':
+			ensure  => file,
+			content => template("candlepin/candlepin.sysconfig.erb"),
+			before	=> Service[$candlepin::service_name],
+		} ~>
+		file { '/etc/systemd/system/candlepin.service':
+			ensure  => file,
+			content => template('candlepin/candlepin.service.erb'),
```

Comment:
This looks like a static file. Perhaps store it in `files/candlepin.service` and call it as `file('candlepin/candlepin.service')`?